rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary can use Windows Management Instrumentation (WMI) to install event filters, providers, consumers, and
+bindings that execute code when a defined event occurs. Adversaries may use the capabilities of WMI to subscribe to an
+event and execute arbitrary code when that event occurs, providing persistence on a system.
+"""
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Windows Management Instrumentation Event Subscription"
```

```suggestion
name = "Persistence via WMI Event Subscription"
```
we use the acronym in other rule names